react/daily-art-prompt/src/components/CommentLayout.js

```diff
@@ -2,14 +2,48 @@ import React from 'react';
 
 import Comment from './Comment.js'
 
-const CommentLayout = ({ comment, onClick }) =>
-    <div>
-        <Comment
-            comment={comment.text}
-        />
-        <button onClick={onClick}>
-            Delete
-        </button>
-    </div >
+const CommentLayout = ({
+    comment,
+    onDelete,
+    onCancel,
+    onEdit,
+    onSubmit
+}) => {
+
+    return (
+        <div>
+            <Comment
+                comment={comment.text}
+            />
+            <div name='nonEditingDiv' className={comment.editing ? 'hidden' : ''}>
```

Comment:
Instead of setting the css hidden attribute here, you could use your `commet.editing` to decide whether you want to display:
1-  `<div name='nonEditingDiv'>` OR
2-  `<div name='editingDiv'>`

Ex: 
```js
{comment.editing ? (
   <div name='nonEditingDiv'> ... </>
) : (
 <div name='editingDiv'> ... </>
)}
```